application/microservices/entando-hub-catalog-ms/src/main/java/com/entando/hub/catalog/rest/BundleGroupVersionController.java

```diff
@@ -115,7 +111,27 @@ public PagedContent<BundleGroupVersionFilteredResponseView, com.entando.hub.cata
         }
 
         logger.debug("Organisation Id: {}, categoryIds {}, statuses {}", organisationId, categoryIds, statuses);
-        return bundleGroupVersionService.searchBundleGroupVersions(sanitizedPageNum, pageSize, Optional.ofNullable(organisationId), categoryIdFilterValues, statuses, searchText);
+        return bundleGroupVersionService.searchBundleGroupVersions(sanitizedPageNum, pageSize, organisationId, categoryIdFilterValues, statuses, searchText);
+    }
+
+    @Operation(summary = "Get all the private bundle group versions in the hub for the selected catalog, provides filter functionality", description = "Public api, no authentication required. You can provide the catalogId, the categoryIds and the statuses [NOT_PUBLISHED, PUBLISHED, PUBLISH_REQ, DELETE_REQ, DELETED]")
+    @GetMapping(value = "catalog/{catalogId}", produces = {MediaType.APPLICATION_JSON_VALUE})
```

Comment:
added new API as the new catalogId parameter would conflict with the organizationId